src/extensions/attributes/index.js

```diff
@@ -30,36 +29,25 @@ function addAttributes( settings ) {
  * @param {function|Component} BlockEdit Original component.
  * @return {string} Wrapped component.
  */
-const withAttributes = createHigherOrderComponent( ( BlockEdit ) => {
-	return ( props ) => {
-		const {
-			attributes,
-			setAttributes,
-		} = props;
-
-		if ( typeof attributes.coblocks === 'undefined' ) {
-			attributes.coblocks = [];
-		}
-
-		//add unique selector
-		if ( allowedBlocks.includes( props.name ) && typeof attributes.coblocks.id === 'undefined' ) {
-			const d = new Date();
-
-			if ( typeof attributes.coblocks !== 'undefined' && typeof attributes.coblocks.id !== 'undefined' ) {
-				delete attributes.coblocks.id;
+const withAttributes = createHigherOrderComponent(
+	BlockEdit => props => {
+		const { name: blockName } = props;
+
+		if ( allowedBlocks.includes( blockName ) ) {
+			props.attributes.coblocks = props.attributes.coblocks || {};
+
+			if ( typeof props.attributes.coblocks.id === 'undefined' ) {
+				const d = new Date();
+				props.attributes.coblocks = Object.assign( {}, props.attributes.coblocks, {
+					id: '' + d.getMonth() + d.getDate() + d.getHours() + d.getMinutes() + d.getSeconds() + d.getMilliseconds(),
+				} );
 			}
-
-			const coblocks = Object.assign( { id: '' + d.getMonth() + d.getDate() + d.getHours() + d.getMinutes() + d.getSeconds() + d.getMilliseconds() }, attributes.coblocks );
-			setAttributes( { coblocks: coblocks } );
 		}
 
-		return (
-			<Fragment>
-				<BlockEdit { ...props } />
-			</Fragment>
-		);
-	};
-}, 'withAttributes' );
+		return <BlockEdit { ...props } />;
```

Comment:
We don't need to wrap the `BlockEdit` component in a `Fragment`, we can just return it.